resources/airbnb.swiftformat

```diff
@@ -22,4 +22,4 @@
 --swiftversion 5.1
 
 # rules
---rules anyObjectProtocol,redundantParens,redundantReturn,redundantSelf,sortedImports,strongifiedSelf,trailingCommas,trailingSpace,wrapArguments,wrapMultilineStatementBraces,indent,wrapAttributes,organizeDeclarations,markTypes,extensionAccessControl,duplicateImports,redundantType
+--rules anyObjectProtocol,redundantParens,redundantReturn,redundantSelf,sortedImports,strongifiedSelf,trailingCommas,trailingSpace,wrapArguments,wrapMultilineStatementBraces,indent,wrapAttributes,organizeDeclarations,markTypes,extensionAccessControl,duplicateImports,redundantType,typeSugar
```

Comment:
Do you think it would be good to always specify the values to SwiftFormat (e.g. `--short-optionals always`) to avoid relying on SwiftFormat defaults which may change? I'm of mixed minds.

Reply:
Good question -- in general I think it makes sense to use SwiftFormat's default options (rather than preferring to always be explicit). There are a fair number of options, which apply to rules we have enabled, that we don't currently have an explicit choice for (and are relying on the default value).